style.css

```diff
@@ -628,71 +705,143 @@ form {
     left: 16%;
   }
 
-  #Projects {
+  /* Work Section */
+
+  .work {
     display: grid;
-    grid-template-rows: repeat(4, 1fr);
-    grid-template-columns: repeat(1fr);
-    place-items: center;
-    margin-top: 100px;
-    gap: 50px;
+    grid-template-rows: repeat(4, auto);
+    column-gap: 24px;
+    row-gap: 142px;
+    margin: 142px;
   }
 
-  .card {
-    display: flex;
+  .project {
+    padding: 24px;
     flex-direction: row;
-    border: 1px solid #dfe1e6;
-    height: 496px;
-    width: 80%;
-    justify-content: center;
-    padding: 16px;
-    border-radius: 16px;
-    background-color: #fff;
+    column-gap: 46px;
+    max-width: 1156px;
+    margin: 0 auto;
   }
 
-  .card-text h2 {
-    font-family: 'Poppins', sans-serif;
-    font-style: normal;
-    font-weight: 700;
+  .row-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .project__image {
+    width: 50%;
+  }
+
+  .project__details {
+    width: 50%;
+  }
+
+  .project__title {
     font-size: 40px;
-    line-height: 52px;
-    margin-bottom: 20px;
-    color: #091e42;
+    line-height: 60%;
+    margin-top: 10px;
+  }
+
+  .project__creator {
+    font-size: 18px;
+    line-height: 133%;
+    margin-bottom: 32px;
+  }
+
+  .project-creator__name {
+    font-weight: 500;
+  }
+
+  .project-creator__details {
+    font-weight: 400;
   }
 
-  .detail {
+  .project__description {
+    font-size: 14px;
+    line-height: 150%;
+    margin-bottom: 24px;
+  }
+
+  .project__description2 {
+    display: block;
+  }
+
+  .project__languages {
     margin-bottom: 40px;
   }
 
-  .right button {
-    margin-left: 0;
+  /* Modal  Section */
+
+  .modal {
+    padding: 1% 8%;
+    overflow-y: hidden;
+  }
+
+  .modal-card {
+    padding: 24px;
+    padding-bottom: 30px;
+    margin: 0 auto;
+    width: 77%;
+    height: 87vh;
+    overflow-y: hidden;
   }
```

Comment:
- [ ] Consider changing the `overflow-y: hidden`  to `overflow-y: scroll` as currently, users are unable to scroll to see overflowing content on the `model` which isn't the best. To create a good user experience it will be nice that you fix this issue so that users are able to read through the full content of the model. This mostly happens between `768px` - `1200px` See the screenshot below for more details:

Figma:
<img width="714" alt="Screenshot 2023-04-20 at 21 27 42" src="https://user-images.githubusercontent.com/107192076/233504549-627b9628-ea55-42c5-885d-b5197f2e635d.png">

Your design:
<img width="1009" alt="Screenshot 2023-04-20 at 22 51 36" src="https://user-images.githubusercontent.com/107192076/233504565-b3ad2396-f966-4695-80ba-7efa04f30c08.png">